### PR TITLE
PHPCS updates

### LINKED ISSUE #1233: Update phpcs.xml to enforce code styling
Currently this codebase has a lot of mixed code styling. Updating the phpcs.xml to reflect the actual style of the codebase should enable contributors to contribute easier and maintain consistent styling moving forward.

### PR DESCRIPTION
**Pull request by CodeProKid**
### Your checklist for this pull request
Thanks for sending a pull request! Please make sure you click the link above to view the contribution guidelines, then fill out the blanks below.

🚨Please review the [guidelines for contributing](CONTRIBUTING.md) to this repository.

- [ ] Make sure you are making a pull request against the **develop branch** (left side). Also you should start *your branch* off *our develop*.
- [ ] Make sure you are requesting to pull request from a **topic/feature/bugfix branch** (right side). Don't pull request from your master!

What does this implement/fix? Explain your changes.
---------------------------------------------------
Made a bunch of updates to the PHPCS file. I disabled a bunch of rules that didn't need to be disabled, and I went through a bunch of them one by one and fixed issues where I thought we should be enforcing rules. This should get us in a much better spot, and we can start deciding what other rules we should start enforcing outside of the WP coding standards.

I also broke out everything into separate commits 😉so you can go through the changes one by one.

Does this close any currently open issues?
------------------------------------------
closes #1233 

Any relevant logs, error output, GraphiQL screenshots, etc?
-------------------------------------
(If it’s long, please paste to https://ghostbin.com/ and insert the link here.)

Any other comments?
-------------------
…

Where has this been tested?
---------------------------
**Operating System:** …

**WordPress Version:** …

### CODE REVIEW

File: src/Data/Connection/AbstractConnectionResolver.php [hunk 1]
```diff
@@ -572,7 +572,7 @@ protected function get_cursor_for_node( $id ) {
 	}
 
 	/**
-	 * get_edges
+	 * Get_edges
```
Comment (jasonbahl) [inline src/Data/Connection/AbstractConnectionResolver.php:575]: what's up with all these changes to ucfirst?

Comment (jasonbahl) [reply src/Data/Connection/AbstractConnectionResolver.php:575]: Actually, we can probably just get rid of the instances of method names at the top of the docblock, eh?

**Comment by jasonbahl** [comment]
@CodeProKid ironically, this PR fails the WPCS check: https://github.com/wp-graphql/wp-graphql/pull/1251/checks?check_run_id=598105292

**Comment by CodeProKid** [comment]
@jasonbahl yeah, it's failing on some code that's in develop but not master, so I can't fix it since it isn't in my branch. I can upstream a fix after this is merged.

File: src/Data/Connection/AbstractConnectionResolver.php (hunk 1 above)
Comment (jasonbahl) [reply src/Data/Connection/AbstractConnectionResolver.php:575]: we can follow up later and remove these

**Comment by jasonbahl** [comment]
@CodeProKid merged.

**Pull request merged**